network-discovery/Makefile

```
@@ -0,0 +1,22 @@
+CGO_ENABLED ?= 0
+BUILD_DIR ?= ./build
+GOARCH ?= $(shell dpkg-architecture -q DEB_BUILD_ARCH)
+GOOS ?= $(shell dpkg-architecture -q DEB_TARGET_ARCH_OS)
```

For more OS agnostic approach, we can utilise `go env`:
```suggestion
GOARCH ?= $(shell go env GOARCH)
GOOS ?= $(shell go env GOOS)
```